seb_openedx/middleware.py

```diff
@@ -22,11 +23,18 @@ def process_view(self, request, view_func, view_args, view_kwargs):
             for permission in self.allow:
                 if permission().check(request, course_key):
                     return
-            return HttpResponseForbidden("Access Forbidden: This course can only be accessed with Safe Exam Browser")
+            return self.render_to_response('seb-403.html', status=403)
 
     @classmethod
     def is_installed(cls):
         """ Returns weather this middleware is installed in the running django instance """
         middleware_class_path = sys.modules[cls.__module__].__name__ + '.' + cls.__name__
         middlewares = settings.MIDDLEWARE_CLASSES if hasattr(settings, 'MIDDLEWARE_CLASSES') else settings.MIDDLEWARE
         return middleware_class_path in middlewares
+
+    def render_to_response(self, template_name, dictionary=None, namespace='main', request=None, **kwargs):
+        """ Custom render_to_response implementation using configurable backend and adding template dir """
+        edxmako = get_edxmako_module()
```

Comment:
This render to response method is what we really want. I suggest instead of having 
`from seb_openedx.edxapp_wrapper.get_edxmako_module import get_edxmako_module` we do 
`from seb_openedx.edxapp_wrapper.edxmako_module import render_to_response` and move all the logic there